pkg/session_recording/recordings.jsx

```diff
@@ -349,10 +420,29 @@
             cockpit.location.go([recording.id], cockpit.location.options);
         }
 
+        componentDidUpdate() {
+            this.drawSortArrow(this.state.sorting_field);
```

Comment:
The `drawSortArrow` function already accesses `this.state.sorting_way`, perhaps we don't need to pass `this.state.sorting_field` to it?